pykafka/balancedconsumer.py

```diff
@@ -208,6 +213,7 @@ def __init__(self,
         self._running = False
         self._worker_exception = None
         self._worker_trace_logged = False
+        self._compacted_topic = compacted_topic
```

Comment:
Boolean attribute should be named as a condition - ie `self._is_compacted_topic`